appyx-navigation/common/src/commonMain/kotlin/com/bumble/appyx/navigation/composable/Children.kt

```diff
@@ -28,12 +28,20 @@ import gestureModifier
 import kotlin.math.roundToInt
 
 @Composable
-inline fun <reified InteractionTarget : Any, ModelState : Any> ParentNode<InteractionTarget>.Children(
+fun <InteractionTarget : Any, ModelState : Any> ParentNode<InteractionTarget>.Children(
     interactionModel: BaseInteractionModel<InteractionTarget, ModelState>,
     modifier: Modifier = Modifier,
     clipToBounds: Boolean = false,
-    noinline block: @Composable ChildrenTransitionScope<InteractionTarget, ModelState>.() -> Unit = {
-        children { child, elementUiModel ->
+    block: @Composable (ChildrenTransitionScope<InteractionTarget, ModelState>.() -> Unit)? = null
```

Comment:
I like it less than default lambda but if we can't make it work I'm okay with it

Reply:
we can always revisit later and see if there's a work-around that JS is happy with.